include/rdma/fabric.h

```diff
@@ -364,6 +382,7 @@ struct fi_ep_attr {
 	size_t			rx_ctx_cnt;
 	size_t			auth_key_size;
 	uint8_t			*auth_key;
+	uint32_t		tclass;
```

Comment:
since we need this as a tx attribute, maybe we should drop it from the ep

Reply:
done